rustfmt.toml

```diff
@@ -4,7 +4,7 @@ use_field_init_shorthand = true
 condense_wildcard_suffixes = true
 normalize_comments = true
 wrap_comments = true
-merge_imports = true
+imports_granularity = "Crate"
```

Comment:
Can we keep unrelated stuff outside of the PR please? I actually started looking into changing our formatting style yesterday. So expect a PR on this soon.

Reply:
Not only because I'm nitpicky ;) but also because I might want to backport this PR. Becomes harder if there are unrelated changes.